spec/exmples/contact_set_spec.rb

```diff
@@ -0,0 +1,32 @@
+describe "ContactSet" do
+    describe "with entries" do
+      before(:all) do
+        @contact_set_json = contact_set_json
+        @contact_set = GoogleContactsApi::ContactSet.new(@contact_set_json)
+      end
+
+      it "should return the right starting index" do
+        expect(@contact_set.start_index).to eq(1)
```

Comment:
```        "openSearch$startIndex": {
            "$t": "1"
        }``` this is what he's getting from contact_set.json line 70, and all the examples in Google's documentation have that value as 1 too